ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -1,79 +1,25 @@
-                <dt class="bold-label"> Date of notification </dt>
-                <dd class="cell-min-height"> @Model.Notification.FormattedSubmissionDate </dd>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <dt class="bold-label"> TB service </dt>
-                <dd> @Model.Notification.TBServiceName </dd>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <dt class="bold-label"> Case manager </dt>
-                <dd class="cell-min-height"> @Model.Notification.Episode.CaseManager </dd>
-            </nhs-grid-column>
-        </nhs-grid-row>
-    </div>
-    <div class="notification-banner-body">
-        <nhs-grid-row classes="notification-overview-details">
-            <nhs-grid-column grid-column-width="OneQuarter" classes="notification-banner-field">
-                <div class="bold-label"> Name </div>
-                <div class="cell-min-height"> @Model.Notification.FullName </div>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <div class="bold-label"> Date of birth </div>
-                <div class="cell-min-height"> @Model.Notification.FormattedDob </div>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <div class="bold-label"> Country of birth </div>
-                <div class="cell-min-height"> @Model.Notification.CountryName </div>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <div class="bold-label"> Postcode </div>
-                <div> @Model.Notification.FormattedNoAbodeOrPostcodeString </div>
-            </nhs-grid-column>
-        </nhs-grid-row>
-        <nhs-grid-row classes="notification-overview-details">
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <dt class="bold-label"> NHS number </dt>
-                <dd class="cell-min-height">  @Model.Notification.FormattedNhsNumber </dd>
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="OneQuarter">
-                <div class="bold-label"> Sex </div>
-                <div class="cell-min-height"> @Model.Notification.SexLabel </div>
-            </nhs-grid-column>
-        </nhs-grid-row> 
-    </div>
-    <div class="notification-banner-triangle"> </div>
-</dl>
-@if(@Model.Notification.NotificationStatus == NotificationStatus.Notified) 
-{
-    <div class="notification-banner-links">
-        <a href="/Notifications/Overview?id=@Model.Notification.NotificationId"> Notification details </a>
+<partial name="./_NotificationDetails" model="@Model.Notification"/>
+
+<div class="notification-banner-links">
+    @if(@Model.Notification.NotificationStatus == NotificationStatus.Notified)
+    {
+        <div class="@((bool)ViewData["IsOverviewPage"] ? "notification-banner-links--current" : "")">

```

Comment:
This wasn't actually part of this ticket, but I added some classes for currently selected pages, and also the class "no-visited-state" to make the header links more inline with the sidebar links